devito/archinfo.py

```diff
@@ -107,15 +113,31 @@ def get_cpu_brand():
         # Fallback 1: it should now be fine to use psutil
         physical = psutil.cpu_count(logical=False)
         if not physical:
-            # Fallback 2: we might end up here on more exotic platforms such a Power8
-            # Hopefully we can rely on `lscpu`
             try:
-                physical = lscpu()['Core(s) per socket'] * lscpu()['Socket(s)']
+                lscpu_mapper = lscpu()
```

Comment:
I just remember that this is memoized, so this extra variable is probably useless. Apologies if I complained about that earlier

Reply:
np